packages/vsx-registry/src/browser/vsx-extension-editor.tsx

```diff
@@ -54,12 +75,34 @@ export class VSXExtensionEditor extends ReactWidget {
         this.updateTitle();
     }
 
+    protected onAfterAttach(msg: Message): void {
+        super.onAfterAttach(msg);
+        this.setContainerHeights();
+    }
+
     protected updateTitle(): void {
         const label = 'Extension: ' + (this.extension.displayName || this.extension.name);
         this.title.label = label;
         this.title.caption = label;
     }
 
+    protected async onResize(msg: Widget.ResizeMessage): Promise<void> {
+        super.onResize(msg);
+        const bodyElement = await this.deferredBodyElementRender.promise;
+        const scrollContainer = await this.deferredScrollContainerRender.promise;
+        if (bodyElement && scrollContainer) {
```

Comment:
it does not look good, since we modify rendered DOM than we should touch only virtual, we should propagate onResize to `VSXExtension` and there update new sizes as part of the state to retrigger DOM update via React

Reply:
I have added an additional method to propagate the `onResize` method from `vsx-extension-editor.tsx` to the `vsx-extension.tsx` and also made changes to ensure only the virtual DOM is manipulated. 

Manipulating the virtual DOM introduced a flickering effect during first render while the scroll container height was being calculated, so I have added some additional functionality to ensure the body is hidden during first render